openedx/core/djangoapps/content/learning_sequences/api/processors/schedule.py

```diff
@@ -58,13 +60,20 @@ def inaccessible_sequences(self, full_course_outline):
         Sequences are inaccessible, regardless of the individual Sequence start
         dates.
         """
+        if self._days_early_for_beta is not None:
+            start_offset = timedelta(days=self._days_early_for_beta)
+        else:
+            start_offset = timedelta(days=0)
+
         # If the course hasn't started at all, then everything is inaccessible.
-        if self._course_start is None or self.at_time < self._course_start:
+        if self._course_start is None or self.at_time < self._course_start - start_offset:
             return set(full_course_outline.sequences)
 
         inaccessible = set()
         for section in full_course_outline.sections:
             section_start = self.keys_to_schedule_fields[section.usage_key].get('start')
+            if section_start is not None:
+                section_start -= start_offset
```

Comment:
Couldn't the section have its own "days_early_for_beta"? I think this field can be specified for all blocks.

Reply:
I asked about that, but while there may have been some miscommunication, I got from the answer that only the course level was to be used.

Reply:
Could you tell me where you asked about that? I cannot see that discussion.

Reply:
@xitij2000 In the upstream ticket. https://openedx.atlassian.net/browse/TNL-7261